Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -148,6 +152,50 @@ def loadSeriesByUID(seriesUIDs):
   loadablesByPlugin, loadEnabled = getLoadablesFromFileLists(fileLists)
   return loadLoadables(loadablesByPlugin)
 
+#------------------------------------------------------------------------------
+def loadByInstanceUID(instanceUID):
+  """ Load with the most confident loadable that contains the instanceUID from DICOM database.
+  This helps in the case where an instance is part of a series which may offer multiple
+  loadables, such as when a series has multiple time points where
+  each corresponds to a scalar volume and you only want to load the correct one.
+  Returns list of loaded node ids (typically one node).
+
+  For example:
+    >>> uid = '1.3.6.1.4.1.14519.5.2.1.3098.5025.172915611048593327557054469973'
+    >>> import DICOMLib
+    >>> nodeIDs = DICOMLib.DICOMUtils.loadByInstanceUID(uid)
+  """
+
+  if not slicer.dicomDatabase.isOpen:
+    raise IOError('DICOM module or database cannot be accessed')
+
+  # get the loadables corresponding to this instance's series
+  filePath = slicer.dicomDatabase.fileForInstance(instanceUID)
+  seriesUID = slicer.dicomDatabase.seriesForFile(filePath)
+  fileList = slicer.dicomDatabase.filesForSeries(seriesUID)
+  loadablesByPlugin, loadEnabled = getLoadablesFromFileLists([fileList,])
```

Comment:
Is the comma needed in `[fileList,]`? I've seen similar syntax at couple of places but normally I don't add comma in such cases and I don't remember having any issues. It is needed when you define a tuple but not for lists, maybe that's the reason? Or is there a special case when the extra comma is useful?

Reply:
You're right - it's only needed for the tuple case and not lists.  I can remove it.